basin3d/synthesis.py

```diff
@@ -337,74 +339,57 @@ def monitoring_features(self, id: str = None, feature_type: str = None, datasour
         USGS-13010450 [(-110.5874305, 43.9038296)]
         ...
 
-        :param id: Unique feature identifier
-        :param feature_type: feature type
-        :param datasource: Datasource id prefix (e.g USGS)
-        :param monitoring_features: List of monitoring feature identifiers (eg. USGS-0010)
-        :param parent_features: List of parent monitoring features to search by
+        :param query:
```

Comment:
Add query definition. From other docsrings: "The query information for this function"

Reply:
Good point.  I thought about that but forgot to update the docs.